index.js

```diff
@@ -0,0 +1,219 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips/';
+
+// Report helpers
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+}
+
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p><ul>`;
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+//LOAD all/specific trips based on url params
+const getTrips = (url) => {
+  const tripList = $('#trip-list');
+  tripList.show();
```

Comment:
This is just a random jQuery thing that I'm not too concerned about but: Since you're calling `.show()` on this element, jQuery actually puts an inline style on this element for `style={display: block;}` ... which is fine, except that it's awkward since there's a class on this called `hidden` that never gets removed... so it ends up looking like this:

```html
<ul class="trips hidden" id="trip-list" style="display: block;"> ... </ul>
```

It's a little awkward to read `class="hidden"` and at the same time see it on the page because of the inline style.

Like I said, I'm not too worried about this... unless it introduces bugs later (what if you try to select on `$('.hidden')`?) It might make sense to conform to one style or the other (either removing the class `hidden` instead of using `.show()`, or in `$(document).ready` calling `$('#trip-list').hide()` right at the beginning